js/renderingPicture.js

```diff
@@ -0,0 +1,37 @@
+import {createPhotos} from './mockPhotos.js';
+
+const dataCreatePicture = createPhotos();
+const SAMPLE = document.querySelector('#picture').content.cloneNode(true);
+const PICTURES = document.querySelector('.pictures');
+const fragment = document.createDocumentFragment();
+
+const createPictures = (data, sample) =>{
+  const sampleClone = sample.cloneNode(true);
+  const randomPhoto = sampleClone.querySelector('.picture__img');
+  randomPhoto.srс = data.url;
+  randomPhoto.alt = data.description;
+  const likes = sampleClone.querySelector('.picture__likes');
+  likes.textContent = data.likes;
```

Comment:
Можно сделать в 1 строчку, сразу достать и установить, но можно оставить как есть
```suggestion
  sampleClone.querySelector('.picture__likes').textContent = data.likes;
```

Reply:
+